freebridge-ai/main.py

```
@@ -1,79 +1,73 @@
-@app.post("/ask")
-async def ask_question(query: Query):
+@app.post("/api/ai/recommend") # 추천 전용 엔드포인트 예시
+async def recommend(query: Query):
     if not rag_chain:
-        raise HTTPException(status_code=503, detail="RAG system is not initialized")
-    
+        raise HTTPException(status_code=503, detail="System not ready")
     try:
         response = rag_chain.invoke({"input": query.question})
-        return {"answer": response["answer"]}
+        return {"answer": response}
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)